extensions/auth-basic/pkg/config/config.go

```diff
@@ -1,23 +1,29 @@
 package config
 
-import "github.com/owncloud/ocis/ocis-pkg/shared"
+import (
+	"context"
+
+	"github.com/owncloud/ocis/ocis-pkg/shared"
+)
 
 type Config struct {
 	*shared.Commons `yaml:"-"`
 	Service         Service  `yaml:"-"`
 	Tracing         *Tracing `yaml:"tracing"`
-	Logging         *Logging `yaml:"log"`
+	Log             *Log     `yaml:"log"`
 	Debug           Debug    `yaml:"debug"`
-	Supervised      bool     `yaml:"-"`
 
 	GRPC GRPCConfig `yaml:"grpc"`
 
 	TokenManager *TokenManager `yaml:"token_manager"`
 	Reva         *Reva         `yaml:"reva"`
 
-	SkipUserGroupsInToken bool          `yaml:"skip_user_groups_in_token"`
+	SkipUserGroupsInToken bool          `yaml:"-"`
```

Comment:
When an admin needs the groups he has to start a custom auth-basic service, right? And there is still a fallback when the groups are needed AFAIR, correct?

Reply:
My thoughts were:
- nobody knows if it's any good
- I don't know someone configuring it
-> don't expose it, since removing this config option is a breaking change. Re-introducing is no breaking change

Reply:
does it disable it? The name `skip` suggests that it will ne skipped if it is set to true 🙈

Reply:
Ah... I think we know someone setting this option to `true`. Currently it defaults to `false` everywhere...

Reply:
The setting was reintroduce